new env setting KX_SSL_VERIFY_SERVER is HOSTIP (#426)
KxSystems/docs: docs/kb/ssl.md

#### SSL_VERIFY_SERVER

Controls the processing of certificates from a server.

Can be set to one of the following values:

-   `YES` (default) checks the X509 certificate the peer presented by verifying the server’s certificate against a trusted source, using the certificates from `SSL_CA_CERT_FILE` or `SSL_CA_CERT_PATH` to verify the server’s certificate. Expired certificates will fail the verification process.

> -   `NO` does not terminate the connection due to a failure verifying the certificate.

Reviewer: ```suggestion
-   `NO` disables certificate verification. The connection will not be terminated if the certificate cannot be verified.
```

> -   `HOSTIP` (since 4.1 2025.11.25, kdbx 5.0 2025.10.18). Same as `YES` setting with additional host and IP verfification. OpenSSL 1.0.2 and above is required. When a hostname is used to connect via TLS/SSL, it checks if the server certificate `SAN` (Subject Alternative Name) or `CN` (Subject CommonName) matches the specified hostname (reference [`X509_check_host`](https://docs.openssl.org/master/man3/X509_check_host/#description)).

Reviewer: ```suggestion
-   `HOSTIP` (since 4.1 2025.11.25. Performs the same verification as the `YES` setting with additional host and IP verification. OpenSSL 1.0.2 and above is required. When a hostname is used to connect via TLS/SSL, it checks if the server certificate `SAN` (Subject Alternative Name) or `CN` (Subject CommonName) matches the specified hostname (reference [`X509_check_host`](https://docs.openssl.org/master/man3/X509_check_host/#description)).
```

> -   `HOSTIP` (since 4.1 2025.11.25. Performs the same verification as the `YES` setting with additional host and IP verification. OpenSSL 1.0.2 and above is required. When a hostname is used to connect via TLS/SSL, it checks if the server certificate `SAN` (Subject Alternative Name) or `CN` (Subject CommonName) matches the specified hostname (reference [`X509_check_host`](https://docs.openssl.org/master/man3/X509_check_host/#description)).

Reviewer: ```suggestion
-   `HOSTIP` (since 4.1 2025.11.25). Performs the same verification as the `YES` setting with additional host and IP verification. OpenSSL 1.0.2 and above is required. When a hostname is used to connect via TLS/SSL, it checks if the server certificate `SAN` (Subject Alternative Name) or `CN` (Subject CommonName) matches the specified hostname (reference [`X509_check_host`](https://docs.openssl.org/master/man3/X509_check_host/#description)).
```
